flask_request_validator/validator.py

```diff
@@ -1,30 +1,49 @@
 import types
-import inspect
 from functools import wraps
 from typing import Tuple
 
 from flask import request
 
-from .exceptions import (
-    NotAllowedType,
-    UndefinedParamType,
-    InvalidRequest,
-    TooManyArguments,
-    InvalidHeader
-)
-from .rules import CompositeRule, ALLOWED_TYPES
+from .exceptions import *
+from .rules import CompositeRule
+from .valid_request import ValidRequest
+from .nested_json import JsonParam
+
 
-# request params. see: __get_request_value()
 GET = 'GET'
 PATH = 'PATH'
 FORM = 'FORM'
-JSON = 'JSON'
 HEADER = 'HEADER'
-PARAM_TYPES = (GET, PATH, JSON, FORM, HEADER)
+_JSON = 'JSON'
```

Comment:
Not sure that is need because we have `JsonParam` now. 
So this is just constant

Reply:
I'll return `JSON` param